radar/languages-and-frameworks/react-native.md

```diff
@@ -5,4 +5,26 @@ quadrant: "languages-and-frameworks"
 tags: [mobile]
 ---
 
-React Native is an open-source mobile application framework created by Facebook. It is used to develop applications for Android, iOS, Web and UWP by enabling developers to use React along with native platform capabilities.
+[React Native](https://reactnative.dev/) is a widely-used open-source mobile application framework developed by Facebook. It's recognized for its utility in building applications for Android, iOS, and other platforms using React along with native platform capabilities. Notably, React Native has been adopted by major technology companies like Microsoft, Amazon, and Sony demonstrating its versatility and reliability in various sectors. This widespread adoption by industry leaders underscores the framework's capability to handle diverse and complex application needs. For more examples of its broad usage, see the [React Native showcase](https://reactnative.dev/showcase).
+
+#### Key Features:
+
+1. **Open Source:** React Native is developed and maintained on [GitHub](https://github.com/facebook/react-native), with a vibrant community contributing to its growth. The framework’s open-source nature allows for transparency, flexibility, and a wide array of contributions from developers worldwide.
+
+2. **Cross-Platform Development:** React Native emphasizes write-once, run-anywhere. Developers can create apps for Android and iOS using the same codebase. This reduces development time and cost, while maintaining performance and a native look and feel.
+
+3. **Live and Hot Reloading:** React Native includes live and hot reloading features, which enhance the development experience by allowing developers to see changes instantly without recompiling the entire app.
+
+4. **Performance:** React Native combines the best aspects of React for the web with native platform capabilities. It delivers high performance for mobile applications by using native components and asynchronous JavaScript interactions.
+
+5. **Community and Ecosystem:** With a vast ecosystem of libraries and tools, React Native offers a robust environment for app development. This ecosystem includes numerous ready-to-use components, which speeds up the development process.
```

Comment:
**issue:** ‏I find little value in having these kinds of considerations - they seem more like a sale pitch from Meta. I'd rather point out what are the **proven** benefits for us.

Reply:
If we have to write only our benefits I agree with you, if we have to extract also some key feature of the tool/language also for the people reading to let them have more insights, maybe not. But It depends on how we should write it.  

Should it be better to point our "best practice" on what to write on these section on the Readme?

Reply:
I simplified the whole card.